include/solvers/heat_transfer_assemblers.h

```diff
@@ -317,8 +318,52 @@ class HeatTransferAssemblerLaser : public HeatTransferAssemblerBase<dim>
 };
 
 /**
- * @brief Class that assembles the laser heat source for the heat
- * transfer solver when VOF is enabled. Exponentially decaying model is
+ * @brief Class that assembles the laser heat source as a surface flux for the
+ * heat transfer solver when VOF is enabled. The laser heat source is only
+ * applied in the metal (when phase value is non-null) using the phase value
+ * alpha as a multiplying factor on the laser heat source.
```

Comment:
what does it mean when the phase value is non-null?

Reply:
Oupss... this is not the appropriate @brief... I'll correct that!